spotify_tensorflow/luigi/tfdv.py

```diff
@@ -84,6 +84,9 @@ def _get_output_args(self):
         stats_path = "{}{}{}{}".format(base_uri.rstrip("/"), sep, file_pattern_dir,
                                        self.stats_file_name)
 
+        # Used in self._publish_outputs
+        self._output_uris = {"output": stats_path}
```

Comment:
I think this makes sense. We don't need to do the typical `generate_uri` thing first because we're just re-using the input path, is that right?